apps/test/unit/turtle/turtleTest.js

```diff
@@ -90,6 +90,25 @@ describe('Artist', () => {
     });
   });
 
+  describe('Accepts a third argument parameter', () => {
+    it('draws a sticker when size is null', () => {
```

Comment:
I think this test case is sufficient for this PR since you're only calling it with null. As you expose more functionality via blocks you can test other cases, like 0 or a positive number.